test/integration/note_test.rb

```diff
@@ -0,0 +1,39 @@
+require 'test_helper'
+
+class PregnancyNoteTest < ActionDispatch::IntegrationTest
+  before do
+    @user = create :user, email: 'testcaseworker@email.com'
+    log_in_as @user
+    @patient = create :patient, name: 'tester', primary_phone: '1231231234'
+    @pregnancy = create :pregnancy, appointment_date: nil, patient: @patient
+    visit edit_pregnancy_path(@pregnancy)
+  end
+ 
+  describe 'add patient pregnancy notes' do
+    it 'should display a pregnancy notes form and current notes' do
+        click_link 'Notes'
+        
+        within('#notes')do
+            assert_text 'Notes' #confirm notes header is visible 
```

Comment:
I expect that the exact attributes we're looking for will get remixed a little when the page gets set up but I think your instinct here is right on!